dashboard/test/ui/features/acquisition_products/curriculum_catalog.feature

```diff
@@ -280,4 +280,103 @@ Feature: Curriculum Catalog Page
+    And the "Grade 10" checkbox is not selected
+    And the "Grade 11" checkbox is not selected
+    And the "Grade 12" checkbox is not selected
+  
+  
+  Scenario: User can use Clear filters button to clear all selected filters
+    Given I am on "http://studio.code.org/catalog"
+    Then I wait until element "#grade-dropdown-button" is visible
+    Then I click selector "#grade-dropdown-button"
+    Then I click selector "span:contains(Kindergarten)"
+    And the "Kindergarten" checkbox is selected
+    Then I wait until element "#duration-dropdown-button" is visible
+    Then I click selector "#duration-dropdown-button"
+    Then I click selector "span:contains(School Year)"
+    And the "School Year" checkbox is selected
+    Then I wait until element "#topic-dropdown-button" is visible
+    Then I click selector "#topic-dropdown-button"
+    Then I click selector "span:contains(Interdisciplinary)"
+    And the "Interdisciplinary" checkbox is selected
+    Then I wait until element "#device-dropdown-button" is visible
+    Then I click selector "#device-dropdown-button"
+    Then I click selector "span:contains(Computer)"
+    And the "Computer" checkbox is selected
+    Then I wait until element "#marketingInitiative-dropdown-button" is visible
+    Then I click selector "#marketingInitiative-dropdown-button"
+    Then I click selector "span:contains(AP CSA)"
+    And the "AP CSA" checkbox is selected
+    Then I wait until element "#clear-filters" is visible
+    Then I click selector "#clear-filters"
+    Then I wait until element "#grade-dropdown-button" is visible
+    Then I click selector "#grade-dropdown-button"
+    And the "Kindergarten" checkbox is not selected
+    Then I wait until element "#duration-dropdown-button" is visible
+    Then I click selector "#duration-dropdown-button"
+    And the "School Year" checkbox is not selected
+    Then I wait until element "#topic-dropdown-button" is visible
+    Then I click selector "#topic-dropdown-button"
+    And the "Interdisciplinary" checkbox is not selected
+    Then I wait until element "#device-dropdown-button" is visible
+    Then I click selector "#device-dropdown-button"
+    And the "Computer" checkbox is not selected
+    Then I wait until element "#marketingInitiative-dropdown-button" is visible
+    Then I click selector "#marketingInitiative-dropdown-button"
+    And the "AP CSA" checkbox is not selected
+
+  @chrome
+  Scenario: User can use Tab navigation on filters, Space to select and escape to close
+    Given I am on "http://studio.code.org/catalog"
+    Then I wait until element "#grade-dropdown-button" is visible
+    Then I click selector "#grade-dropdown-button"
+    Then I click selector "span:contains(Kindergarten)"
+    And I press keys ":tab"
+    And I press keys ":space"
+    And the "Grade 1" checkbox is selected
+    And I press keys ":escape"
+    Then I wait until element ".dropdown-menu" is not visible
+
```

Comment:
Nit: remove the chunk of whitespace at the bottom here, leaving a single newline